plugins/services/images/local.go

```diff
@@ -214,15 +211,3 @@ func (l *local) Delete(ctx context.Context, req *imagesapi.DeleteImageRequest, _
 
 	return &ptypes.Empty{}, nil
 }
-
-func (l *local) emitSchema1DeprecationWarning(ctx context.Context, image *images.Image) {
-	if image == nil {
-		return
-	}
-	dgst, ok := image.Labels[images.ConvertedDockerSchema1LabelKey]
```

Comment:
`images.ConvertedDockerSchema1LabelKey` should no longer be referenced.  Can you remove it?

Reply:
The PR no longer removes Schema 1